src/Aspire.Dashboard/Components/Controls/ResourceSelectOptionTemplate.razor

```diff
@@ -0,0 +1,22 @@
+﻿@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Model.Otlp
+
+<FluentStack Orientation="Orientation.Horizontal" HorizontalGap="3">
+    @if (ViewModel.Id.Type is OtlpApplicationType.ReplicaSet)
+    {
+        <FluentIcon Icon="Icons.Regular.Size20.AppsList"
+                    Color="Color.Accent"
+                    Style="vertical-align: text-bottom;"/>
+    }
+    else if (ViewModel.Id.Type is OtlpApplicationType.Replica)
+    {
+        <div style="width: 20px;"></div>
+    }
+
+    @ViewModel.Name
+</FluentStack>
+
+@code {
+    [Parameter, EditorRequired]
+    public required SelectViewModel<(OtlpApplicationType? Type, string? InstanceId)> ViewModel { get; set; }
```

Comment:
Can you add a class or record instead of the tuple? Tuples are great, but if you need to change them, then you have to change them everywhere, and this tuple is used in a lot of places. A class or record centralizes the definition.

Reply:
added a record!